1_fetch/src/download_data.R

```diff
@@ -24,5 +29,5 @@ download_nwis_data <- function(ind_file, site, dates) {
   # Write the data file and the indicator file
   data_file <- scipiper::as_data_file(ind_file)
   saveRDS(nwis_data, data_file)
-  sc_indicate(ind_file) # just until s3_put is added
+  s3_put(ind_file) 
```

Comment:
My only question is whether this needs to be `s3_put(ind_file, data_file)`, but I'm not sure.

Reply:
No, this works when you use `s3_put` in the same function that creates that data file (per the `s3_put` docs)